.github/workflows/ci.yml

```diff
@@ -183,28 +185,32 @@ jobs:
         env:
           PYSYC_DOC_BUILD_VERSION: 23_1
 
+      - name: Build PDF Documentation
+        run: |
+          sudo apt update
+          sudo apt-get install -y texlive-latex-extra latexmk
+          make -C doc pdf SPHINXERRWARN=""
+        env:
+          PYSYC_DOC_BUILD_VERSION: 23_1
+          extra_mem_bot: 10000000
+          extra_mem_top: 15000000
```

Comment:
What are these env variables doing?

Reply:
These are for the pdf build. it was failing to make pdf because of the memory size. This is giving a higher limit of memory in latexmk.